WalletWasabi/Tor/Socks5/TorSocks5Client.cs

```diff
@@ -363,6 +364,19 @@ private async Task<byte[]> SendAsync(byte[] sendBuffer, int? receiveBufferSize =
 					// Receive the response
 					var receiveBuffer = new byte[actualReceiveBufferSize];
 
+					// Read exactly "receiveBufferSize" bytes.
+					if (receiveBufferSize != null)
```

Comment:
Isn't this `actualReceiveBufferSize`?

Reply:
No, it's meant to be `receiveBufferSize`. Meaning, when you specify `receiveBufferSize` in the `SendAndReceiveAsync` method parameter, you want to read exactly that number of bytes. Otherwise, you don't know and other mechanisms need to be used (This is currently slightly buggy and that's what the next PR will be about).

Reply:
ACK